### ISSUE TITLE
Feature Request: Add FAQ/Commenting to Card Detail Page

### ISSUE THREAD

**Issue by zliniguez** [fafranco82/swdestinydb#131]
# Summary
* When viewing an individual card page it would be extremely helpful and useful for players to allow for comments, particularly in the format of a wiki FAQ. In this way the community can provide clarification for common rules questions. Bonus points for easy-to-remember urls (e.g. swdestinydb.com/cards/secondchance

**Comment by fafranco82** [comment]
Indeed, you can write a review/comment on a card currently. There is a button just below the card image:

![image](https://cloud.githubusercontent.com/assets/5708531/26714997/3be2b51e-4774-11e7-95bf-cbc9cd3f04b6.png)

About FAQ, and user sent me a spreadsheet with a lot of FAQ content that I want to export to the database ;)

**Comment by zliniguez** [comment]
Yes! Having a legitimate wiki-type FAQ section would be insanely helpful. I ask because I'm the admin of the Facebook group, and having a huge resource like swdestinydb.com to point user questions to would be extremely useful.

**Comment by piccolereti** [comment]
hi, if you can improve, whit some another filters that show , cards not owned,  owned only 1, owned up 2 , is good for fast trade to know which ones you need, or want sell , thank you so much

**Comment by zliniguez** [comment]
Hi! Can we pursue this item further? The SWD Facebook Group gets a TON of rules questions, and having an additional FAQ/Rulings section (with voting?) would be immensely helpful to players.
